k8_cortx_cloud/logs-cortx-cloud.sh

```diff
@@ -103,7 +132,14 @@ function getInnerLogs()
       --cluster_conf_path yaml:///etc/cortx/cluster.conf \
       --location file://${path} \
       --bundle_id "${name}" \
-      --message "${name}"
+      --message "${name}" \
```

Comment:
Just noticed this is set to an empty string. Is that correct?

Reply:
what exactly set to empty? @rohit-k-dwivedi plz follow up and fix if needed for anything